wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/RpcElement.java

```diff
@@ -66,6 +76,8 @@ public final String toSchema() {
     Builder requestType(String requestType);
     Builder responseType(String responseType);
     Builder options(ImmutableList<OptionElement> options);
+    Builder clientStreaming(boolean clientStreaming);
```

Comment:
They could, I was just copying the name from MethodDescriptorProto. I agree requestStreaming and responseStreaming would be clearer.